### PR TITLE
JBPM-6487 Jbpm tests stabilisation

### PR DESCRIPTION
**Pull request by dcihak**
Fixes IntermediateEventTest, TaskCommentTest and TaskExpirationDateTest

### CODE REVIEW

**Comment by dcihak** [comment]
[DO NOT MERGE]

File: jbpm-bpmn2/src/test/java/org/jbpm/bpmn2/IntermediateEventTest.java [hunk 1]
```diff
@@ -1206,7 +1206,7 @@ public void testTimerBoundaryEventInterruptingOnTaskCancelTimer() throws Excepti
         ksession = restoreSession(ksession, true);
         ksession.getWorkItemManager().registerWorkItemHandler("Human Task", handler);
         timers = getTimerManager(ksession).getTimers();
-        assertEquals(0, timers.size());
+        assertTrue(timers == null || timers.isEmpty());
```
Comment (rsynek) [inline jbpm-bpmn2/src/test/java/org/jbpm/bpmn2/IntermediateEventTest.java]: I would rather avoid using assertTrue (if you are not asserting a value of boolean variable, of course). Reason is that when assertTrue fails, it does not tell you why - was is null or empty? Please take a look at assertJ library, is provides decent asserts for (not only) collections.

File: jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/TaskCommentTest.java [hunk 2]
```diff
@@ -43,9 +30,19 @@
 import org.kie.internal.task.api.model.InternalComment;
 import org.kie.internal.task.api.model.InternalOrganizationalEntity;
 
+import javax.persistence.EntityManagerFactory;
```
Comment (rsynek) [inline jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/TaskCommentTest.java]: Is this change of imports based on KIE group code style? If not, I would rather avoid unnecessary formatting changes like these.

**Comment by dcihak** [comment]
Imports formatting and empty collection assert fixed. If OK, READY TO BE MERGED.
